cypress.config.mjs

```diff
@@ -2,8 +2,18 @@ import { defineConfig } from 'cypress'
 
 export default defineConfig({
   e2e: {
+    chromeWebSecurity: false,
     supportFile: false,
     defaultCommandTimeout: 10000,
-    screenshotOnRunFailure: (process.env.CI !== 'true')
+    screenshotOnRunFailure: (process.env.CI !== 'true'),
+    setupNodeEvents(on, config) {
+      on('task', {
+        log(message) {
+          console.log(message)
+
+          return null
+        },
+      })
+    }
```

Comment:
❓ what's this needed for?

Reply:
Oh this allows you to do some basic debugging from the command line bc I think if you console log in a cypress test it prints it in the fake browser console not the command line

Reply:
Lol I'm a bit of a console log debugger 😅

Reply:
No worries, just curious 😅